docs/02-app/01-building-your-application/07-configuring/05-mdx.mdx

```diff
@@ -65,20 +65,20 @@ This allows `.md` and `.mdx` files to act as pages, routes, or imports in your a
 Create a `mdx-components.tsx` (or `.js`) file in the root of your project to define global MDX Components. For example, at the same level as `pages` or `app`, or inside `src` if applicable.
 
 ```tsx filename="mdx-components.tsx" switcher
-import type { MDXComponents } from 'mdx/types'
+const components = {}
```

Comment:
should all of the creations of `components` objects in this PR also have `satisfies MDXComponents` after the object?

eg.

```tsx
import type { MDXComponents } from 'mdx/types'

const components = {} satisfies MDXComponents
```